src/StripeTests/Infrastructure/TestData/TestOptions.cs

```diff
@@ -43,6 +43,12 @@ public enum TestEnum
         [JsonProperty("metadata")]
         public Dictionary<string, string> Metadata { get; set; }
 
+        [JsonProperty("list_of_doubles")]
+        public List<double> ListOfDoubles { get; set; }
```

Comment:
flagging: what about decimals?

Reply:
Ah right, we use `decimal` in the library, not `double`. I'll change the test. (Though note that the encoding treats all scalars the same way -- it simply calls `ToString()` on them.)